qcodes/instrument_drivers/AlazarTech/ATS9440.py

```diff
@@ -0,0 +1,328 @@
+from .ATS import AlazarTech_ATS
+from .utils import TraceParameter
+from qcodes.utils import validators
+
+
+class AlazarTech_ATS9440(AlazarTech_ATS):
+    """
+    This class is the driver for the ATS9440 board
+    it inherits from the ATS base class
+    """
+    samples_divisor = 256 #32
+
+    def __init__(self, name, **kwargs):
+        dll_path = 'C:\\WINDOWS\\System32\\ATSApi.dll'
+        super().__init__(name, dll_path=dll_path, **kwargs)
+
+        # add parameters
+
+        # ----- Parameters for the configuration of the board -----
+        self.add_parameter(name='clock_source',
+                           parameter_class=TraceParameter,
+                           get_cmd=None,
```

Comment:
set_cmd=None can be added to this parameter.

Reply:
Done